src/PowerSystems2PRAS.jl

```diff
@@ -396,33 +302,29 @@ function generate_pras_system(
     λ_gen = Matrix{Float64}(undef, n_gen, s2p_meta.N)
     μ_gen = Matrix{Float64}(undef, n_gen, s2p_meta.N)
 
+    #FIXME
```

Comment:
@josephmckinsey is this pending? Do we need to add an issue or a comment to this?

Reply:
Yes. This needs to be fixed but this logic works for now. We will implement the component map solution we discussed for this.

Reply:
open an issue for this please

Reply:
https://github.com/NREL-Sienna/SiennaPRASInterface.jl/issues/60

Reply:
Tracked in #60